### DISCUSSION TITLE
ESM-2 parameter initialization

### DISCUSSION THREAD

**Discussion by eric-tc-wong** [facebookresearch/esm#276] [Q&A] [answered]
Hello,

Was ESM-2 initialized the same way as ESM-1b? I am wondering if a different initialization method may have contributed to the impressive improvement of ESM-2 over ESM-1b.

Thanks,
Eric

**Comment by tomsercu** [comment, marked as answer]
We didn't make changes to the initialization for ESM-2.
Correct me if I'm wrong @halilakin

**Reply by eric-tc-wong** [reply]
Thank you for your reply. When loading the model data I saw the flag "use_bert_init=False" and checked that "use_bert_init=True" in the earlier model. Perhaps that refers to something else.

Regards,
Eric
